docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,132 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
```

Comment:
@madanadit Since there's no existing upgrade guide I can refer to, I had to create an issue instead. [Request for an upgrade guide](https://github.com/Alluxio/alluxio/issues/10002)

There are a few options how we can handle this:

1. Cover the things we need to cover (some may better belong elsewhere), remove them once the upgrade guide doc is ready.

2. Put this doc on hold, wait for the blocking transaction to finish and then continue.

3. Leave a placeholder link, either to an non-existent page or just blank link waiting to be updated.

Reply:
Option 1 sgtm. Include minimal info (k8s specific only when possible) in this doc